plugin/storage/kafka/options.go

```diff
@@ -43,13 +44,19 @@ const (
 	suffixCompression      = ".compression"
 	suffixCompressionLevel = ".compression-level"
 	suffixProtocolVersion  = ".protocol-version"
+	suffixBatchLinger      = ".batch-linger"
+	suffixBatchSize        = ".batch-size"
+	suffixBatchMaxMessages = ".batch-max-messages"
 
 	defaultBroker           = "127.0.0.1:9092"
 	defaultTopic            = "jaeger-spans"
 	defaultEncoding         = EncodingProto
 	defaultRequiredAcks     = "local"
 	defaultCompression      = "none"
 	defaultCompressionLevel = 0
+	defaultBatchLinger      = time.Duration(0 * time.Millisecond)
```

Comment:
this should also stay 0.

I would avoid using constants for this, just use `0` literal, which makes it obvious when the flag is declared that it has no default value.

Reply:
Changed to 0